homeassistant/components/hive/light.py

```diff
@@ -12,19 +12,18 @@
 )
 import homeassistant.util.color as color_util
 
-from . import ATTR_AVAILABLE, ATTR_MODE, DATA_HIVE, DOMAIN, HiveEntity, refresh_system
+from . import HiveEntity, refresh_system
+from .const import ATTR_AVAILABLE, ATTR_MODE, DOMAIN
```

Comment:
What does the `ATTR_MODE` attribute measure?

Reply:
Hive has a concept of modes alongside the normal functionality.

This attribute returns a string representing the mode the device is in, its used as a device state attribute. e.g MANUAL, SCHEDULE